elastic/integrations
# Update README files using the url placeholder for elastic documentation links

## What does this PR do?

Update README files to use the new `{{ url <key> <link> }}` notation introduced in https://github.com/elastic/elastic-package/pull/938

To accomplish that, a new file in the root of the repository `links_table.yml` has been added with all the links to elastic documentation found in the current packages.

In general, there are no changes in README contents with this PR. There are some changes due to some markdown syntax issues.

## Checklist



## How to test this PR locally

- In elastic-package repository, Build elastic-package command from the branch in the PR (`make build`)
- In integrations repository, go to any of the packages updated and run `elastic-package build`. This command should generate the README without failures.

## Author's Checklist

- [x] Requires a new version of elastic-package integrated including https://github.com/elastic/elastic-package/issues/948

## Related issues


- Requires https://github.com/elastic/elastic-package/issues/948
- Closes https://github.com/elastic/integrations/issues/4067
- Part of https://github.com/elastic/integrations/issues/2652
